app/uk/gov/hmrc/testuser/services/Generator.scala

```diff
@@ -19,7 +19,7 @@ package uk.gov.hmrc.testuser.services
 import org.scalacheck.Gen
 import uk.gov.hmrc.domain._
 import uk.gov.hmrc.testuser.models.ServiceName._
-import uk.gov.hmrc.testuser.models.{MtdItId, TestAgent, TestIndividual, TestOrganisation}
+import uk.gov.hmrc.testuser.models.{ServiceName => _, _}
```

Comment:
It looks like it is a thing - http://docs.scala-lang.org/cheatsheets/
It imports everything from the package except the things that are filtered.